bentoml/_internal/io_descriptors/json.py

```diff
@@ -38,7 +38,7 @@
     )
 
 
-JSONType = t.Union[str, t.Dict[str, t.Any], "pydantic.BaseModel"]
+JSONType = t.Union[str, t.Dict[str, t.Any], "pydantic.BaseModel", None]
```

Comment:
```suggestion
JSONType = t.Optional[str, t.Dict[str, t.Any], "pydantic.BaseModel"]
```

Reply:
Optional seems to take only one input.
```
TypeError: Optional[t] requires a single type. Got (<class 'str'>, typing.Dict[str, typing.Any], 'pydantic.BaseModel').
```

Reply:
Oh my bad, it has to be `t.Optional[t.Union[...]]`

Reply:
but `t.Union[.., None]` works as well.